python/rascaline/rascaline/systems/__init__.py

```diff
@@ -32,6 +33,15 @@ def __init__(self):
     .. _chemfiles.Frame: http://chemfiles.org/chemfiles.py/latest/reference/frame.html
     """
 
+if HAVE_PYSCF:
```

Comment:
Could you add pyscf to the `docs` env in tox.ini, to make sure this is included in the documentation?

Reply:
It would be nice to also add a test (and add pyscf in the `all-deps` env in tox) checking that you get the right data out of both PySCF classes, for both molecules and periodic systems.